style.css

```diff
@@ -0,0 +1,452 @@
+*{
+    box-sizing: border-box
+}
+
+body{
+    font-family: Helvetica, Arial, sans-serif;
+    line-height: 20px;
```

Comment:
Why did you set a line height of 20px? You are just making your life harder because if you change the size of fonts, they will be bigger than their content and overflow.

Reply:
mean but true :D